test/time_tracker/test_helpers.clj

```diff
@@ -32,19 +33,34 @@
   [channel]
   (async/<!! channel))
 
+(defn read-dup
+  []
+  (let [dup (chan 1)]
+    (async/go-loop []
+      (log/info (str "Read " (async/<! dup) " from duplicate channel"))
+      (recur))
+    dup))
+
 (defn make-ws-connection
   "Opens a connection and completes the auth handshake."
   [google-id]
+
   (let [response-chan (chan 5)
-        socket        (ws/connect "ws://localhost:8000/api/timers/ws-connect/"
-                                  :on-receive #(put! response-chan
-                                                     (json/decode % keyword)))]
-    (ws/send-msg socket (json/encode
+        dup-chan (read-dup)
+        conn (ws/connect "ws://localhost:8000/api/timers/ws-connect/"
+                         :on-receive #(doseq [c [response-chan dup-chan]]
```

Comment:
If all you're doing is logging the data received, you don't need a different `core.async` channel for this.